src/h2load.cc

```diff
@@ -366,9 +438,16 @@ Client::~Client() {
     worker->sample_client_stat(&cstat);
   }
   ++worker->client_smp.n;
+  
+  if (!measurement_calculation_done) {
+    std::cout << "Duration timeout not called on Client: " << id << std::endl;
```

Comment:
Is this a warning? An error? Should it be fatal? Is it a programmer bug or something that can happen in other ways?